cmd/containerd-stargz-grpc/main.go

```diff
@@ -57,21 +59,30 @@ const (
 	defaultLogLevel            = logrus.InfoLevel
 	defaultRootDir             = "/var/lib/containerd-stargz-grpc"
 	defaultImageServiceAddress = "/run/containerd/containerd.sock"
+	defaultFuseManagerAddress  = "/run/containerd-stargz-grpc/stargz-fuse-namanger.sock"
```

Comment:
`/run/containerd-stargz-grpc/fuse-manager.sock` according to `fusermanager/fusemanager.go`.